### ISSUE TITLE
Sharing Settings: Move "Add more" button into "Edit sharing buttons"

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#32226]
<img width="1437" alt="Screen_Shot_2019-03-06_at_10 35 32" src="https://user-images.githubusercontent.com/2124984/55990347-7b3c7700-5c75-11e9-9a4f-4f882a5260d6.png">

Problem: The "Add "more" button" adds clutter to the screen and can be moved within the edit sharing buttons as an another option.

Steps to reproduce:
1/ visit sharing > sharing buttons

**Comment by alaczek** [comment]
Agreed this looks cluttered, and since the "drawer" with social media services that shows up when you click either button looks identical, it can be hard to tell the difference without carefully reading the description. 
I'm wondering though if tucking "More" button config under single "Edit" button won't make it more difficult to configure. I guess it depends on how we implement it - did you have any ideas around that?

**Comment by scalarbane2** [comment]
Hello all, I tried three different ways we can represent this. Let me know your thoughts on this and the best way to move forward.

![Add _More_-1](https://user-images.githubusercontent.com/50745054/58272338-2fa5de80-7d86-11e9-9c18-71f694ee79ca.png)

![Add _More_-2](https://user-images.githubusercontent.com/50745054/58272340-2fa5de80-7d86-11e9-8e87-464161cc2605.png)

![Add _More_](https://user-images.githubusercontent.com/50745054/58272341-303e7500-7d86-11e9-8e95-c47eb53a0e91.png)

**Comment by sixhours** [comment]
I think the above is a good starting point! 

Gideon and I talked about this in Slack, and he brought up a good point, that it would be good to have some data on the More button before we proceed. I pinged Robert F. to get an idea of whether we can/should track data on the use of the More button specifically, since he recently added tracking information for the Edit button.

**Comment by sixhours** [comment]
Checked with Robert and he confirmed that it's OK if we add a separate Tracks event for the More button. That should be our next step here.

**Comment by sixhours** [comment]
Now that we have Tracks flags added, I'll move this to On Hold and we can wait a couple weeks to see if anyone is using this button.

**Comment by sixhours** [comment]
It's been a month since we added the differentiated Tracks flags, so I checked on the stats; about 16% of folks who view the Sharing Buttons section click the "Add More button" button and save their settings. The overall number of daily clicks on that button is around 200 uniques. 

With that in mind, I think it's safe to move under the "Edit" button without too much disruption. What do y'all think?

**Comment by sixhours** [comment]
It may also be worth waiting until after we've conducted a Calypso UX audit to make any decisions, since this flow could qualify for an audit and get redesigned entirely; thoughts @lcollette ?
